infra/scoped/lambda.tf

```diff
@@ -96,10 +96,10 @@ resource "aws_lambda_function" "api" {
       AUTH0_CLIENT_ID      = auth0_client.api_gateway_identity.client_id,
       AUTH0_CLIENT_SECRET  = auth0_client.api_gateway_identity.client_secret,
       API_ALLOWED_ORIGINS  = local.identity_v1_docs_endpoint,
-      EMAIL_SMTP_HOSTNAME  = aws_ssm_parameter.email_smtp_hostname.value,
-      EMAIL_SMTP_PORT      = aws_ssm_parameter.email_smtp_port.value,
-      EMAIL_SMTP_USERNAME  = aws_ssm_parameter.email_smtp_username.value,
-      EMAIL_SMTP_PASSWORD  = data.aws_secretsmanager_secret_version.email_smtp_password_secret_version.secret_string,
+      EMAIL_SMTP_HOSTNAME  = local.email_credentials["smtp_hostname"],
+      EMAIL_SMTP_PORT      = local.email_credentials["smtp_port"],
+      EMAIL_SMTP_USERNAME  = local.email_credentials["smtp_username"],
+      EMAIL_SMTP_PASSWORD  = local.email_credentials["smtp_password"],
```

Comment:
Not to address here, but I'd quite like us to get away from passing credentials in Lambda environment variables, and put them in Secrets Manager instead.

Reply:
I think this would require (a) adding permissions for this lambda to read credentials out of SM in the right place, and (b) a change to the lambda code to read from there and decode the creds.

I agree handing secrets around in this case is unsatisfactory.